src/packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -158,9 +158,29 @@ get ${methodName}(): Prisma.${m.model}Delegate;`
   public toTS(): string {
     return `${new Datasources(this.internalDatasources).toTS()}
 
+export type RejectOnNotFound = boolean | Error | ((error: Error) => Error)
+type LooseRecord<K extends string | number | symbol, T> = { [P in K]?: T; }
```

Comment:
Can you elaborate why we need a `LooseRecord`?

Reply:
Not needed so i've removed it